src/atmos_param/two_stream_gray_rad/two_stream_gray_rad.F90

```diff
@@ -526,6 +525,7 @@ subroutine two_stream_gray_rad_down (is, js, Time_diag, lat, lon, p_half, t,
 case(B_GEEN)
   ! split LW in 2 bands: water-vapour window and remaining = non-window
   ! ref: Ruth Geen etal, GRL 2016 (supp. information).
+  lw_dtrans_win = 1.
```

Comment:
if the `lw_dtrans_win` variable is going to be used by other radiation schemes, the alternative fix to this would be to move line 278 out of the `case(B_GEEN)` guard